python/ray/_private/runtime_env/pip.py

```diff
@@ -132,9 +137,8 @@ def _create_or_get_virtualenv(cls, path: str, cwd: str, logger: logging.Logger):
         python = sys.executable
         virtualenv_path = os.path.join(path, "virtualenv")
         virtualenv_app_data_path = os.path.join(path, "virtualenv_app_data")
-        current_python_dir = os.path.abspath(
-            os.path.join(os.path.dirname(python), "..")
-        )
+
+        current_python_dir = sys.prefix
```

Comment:
Use the canonical method to find the top-level python directory. On windows `python.exe` can be either in the top-level directory or in `Scripts`, so using `sys.prefix` works around complicated logic.

Reply:
moved this to windows-only to make sure it does not break non-windows